backend/src/main/scala/bloop/tracing/BraveTracer.scala

```diff
@@ -77,7 +111,8 @@ object BraveTracer {
   val zipkinServerUrl = Option(System.getProperty("zipkin.server.url")).getOrElse(
     "http://127.0.0.1:9411/api/v2/spans"
   )
-  val debugTrace = Properties.propOrFalse("zipkin.trace.debug")
+  val isDebugTrace = Properties.propOrFalse("zipkin.trace.debug")
```

Comment:
Although this line is unrelated to the PR I think it's good to also use a `bloop.` prefix for consistency with `bloop.trace.verbose`
```suggestion
  val isDebugTrace = Properties.propOrFalse("bloop.trace.debug")
```

Reply:
done!